wiki/Mapping_and_Modding_Timeline/en.md

```diff
@@ -0,0 +1,302 @@
+
+### June
+
+- **2018-06-21:** The [Quality Assurance Helper](/wiki/People/The_Team/Beatmap_Nominators/General_Information#quality-assurance-helpers) initiative is created to ensure all qualified beatmaps are reviewed by at least 2 BN or QAT before rank. The program was managed via Trello. <!-- https://osu.ppy.sh/community/forums/topics/764134-->
+
+### July
+
+- **2018-07-29:** New beatmap spread rules are implemented, completely reshaping how many difficulties are created in ranked mapsets going forward. <!-- https://osu.ppy.sh/community/forums/topics/726474 -->
+
+### December
+
+- **2018-12-12:** The osu!team takes a [negative stance on beatmap commissions](https://osu.ppy.sh/community/forums/topics/840838), acknolwedging that they will happen, but pledging to take no action unless members of the BN or QAT are involved.
+- **2018-12-17:** [-Mo-](https://osu.ppy.sh/users/2202163) becomes a QAT leader after [JBHyperion](https://osu.ppy.sh/users/4879508) steps down.
+- **2018-12-17:** The QAT is declared as having no future in its current state and ideas for reform are encouraged. <!-- https://osu.ppy.sh/community/forums/topics/842601 -->
+
+**2749 beatmaps** were ranked in 2018.
+
+## 2019
+
+### January
+
+- **2019-01-29:** A proposal for QAT reform is created as a conglomeration of other users' proposals. This introduces the concept of the [Nomination Assessment Team (*NAT*)](/wiki/People/The_Team/Nomination_Assessment_Team). <!-- https://osu.ppy.sh/community/forums/topics/862002 -->
+
+### March
+
+- **2019-03-01:** The [Mappers' Guild](/wiki/Mappers_Guild) transitions from application-based entry to automatic entry for anyone with 3 ranked maps. [Featured Artist](/wiki/Featured_Artists) beatmaps are more frequently created and ranked. `#mappers-guild` channel is created in the [osu!dev Discord server](/wiki/osu!dev_Discord_server).
+- **2019-03-13:** Rule requiring consecutive nominations to be set at least 24 hours apart from each other is removed. <!-- https://osu.ppy.sh/community/forums/posts/6885033 -->
+
+### May
+
+- **2019-05-05:** The BN/QAT restructure is finalized. The Quality Assurance Team becomes the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team), while the Beatmap Nominators become responsible for disqualifications and veto mediation. The [Beatmap Nominator website](https://bn.mappersguild.com/) was published to automate many BN/NAT tasks.
+
+### August
+
+- **2019-08-05:** [Naxess](https://osu.ppy.sh/users/8129817) releases the Mapset Verifier <!-- TODO: stub --> tool, automating a significant portion of beatmap checking processes for Beatmap Nominators.
+- **2019-08-06:** Beatmap Nominators are able to participate in BN application evaluations. Their input is used to determine applicants and to judge potential new NAT additions.
+
+### October
+
+- **2018-10-22:** [Ascendance](https://osu.ppy.sh/users/2931883) becomes a NAT leader after [Mao](https://osu.ppy.sh/users/2204515) steps down.
```

Comment:
this type of thing is good to document as history ofc but I feel like it also needs to be connected to other points in this document or at least commented on why it's significant. (for this line) what did Ascendance do as NAT leader? did he have an impact on modding and mapping worth writing down? this doesn't tell reader anything important and it's not easy to discover answers to these questions if you weren't aroudn at the time